test/unit/about/welcomePageTest.js

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+/* global describe, before, after, it */
+
+const mockery = require('mockery')
+const {shallow} = require('enzyme')
+const assert = require('assert')
+let AboutWelcome
+require('../braveUnit')
+
+describe('AboutWelcome component', function () {
```

Comment:
Is this intentional? The test plan says `npm run test -- --grep="about:welcome"`, which does not trigger that.